config/nrfconnect/chip-module/Kconfig.defaults

```diff
@@ -288,40 +289,38 @@ config MBEDTLS_ENABLE_HEAP
     default y
 
 config MBEDTLS_HEAP_SIZE
-    default 15360
-
-config MBEDTLS_TLS_LIBRARY
-    default y
+    default 15360 if !CHIP_CRYPTO_PSA
+    default 32768 if CHIP_CRYPTO_PSA
 
 config NRF_SECURITY_ADVANCED
-    default y
+    default y if !CHIP_CRYPTO_PSA
 
 config MBEDTLS_AES_C
-    default y
+    default y if !CHIP_CRYPTO_PSA
 
 config MBEDTLS_ECP_C
-    default y
+    default y if !CHIP_CRYPTO_PSA
```

Comment:
Don't we need to enable any PSA_WANT_ALG_XXX here? We may rely on OT here, but it would be good to have them enabled directly in Matter as well to support Wi-Fi and possibly other platforms in the future as well.